Marlin/temperature.cpp

```diff
@@ -1544,70 +1546,79 @@ void Temperature::isr() {
   #endif
 
   #if DISABLED(SLOW_PWM_HEATERS)
```

Comment:
The `ENABLED` macro contains a `defined()` expression which cannot be included anywhere except in a preprocessor expression.